samples/kubernetes/helm/fhir-server/templates/deployment.yaml

```diff
@@ -35,6 +35,10 @@ spec:
           env:
             - name: ASPNETCORE_FORWARDEDHEADERS_ENABLED
               value: "true"
+            {{- if .Values.convertData.enabled }}
+            - name: FhirServer__Operations__ConvertData_Enabled
```

Comment:
Should there be a double underscore between ConvertData and Enabled instead of a single one?

Reply:
Oh! You're right.
I should have copied the parameter from the document instead of typing it literally.
I'll push a new commit to fix this.